src/util/AppContextUtil.tsx

```diff
@@ -73,22 +74,54 @@ class AppContextUtil {
     state.mapView.present.zoom = mapConfig.zoom;
 
     // mapLayers
-    state.mapLayers = union(state.mapLayers, mapLayers);
-    state.mapLayers = AppContextUtil.parseLayers(mapLayers);
+    state.mapLayers = AppContextUtil.parseLayertree(layerTree);
```

Comment:
will this break the existing approach with flat layer hierarchie? (lkeyword: ayerset / layer topic)

Reply:
to be honest i did not check. i checked the default `static` application and fixed the new shogun-webapp one.
We have an unclear situation here as one can configure layertree and topics at the same time. We need to distinguish that better and need a bigger discussion about all this.